Formula/php.rb

```diff
@@ -0,0 +1,258 @@
+class Php < Formula
+  desc "General-purpose scripting language"
+  homepage "https://php.net"
+  url "https://github.com/php/php-src/archive/php-7.1.9.tar.gz"
+  sha256 "4bb7acacee5034705673004010789d6313e7d7f490a270308ec75d3391c4afea"
+
+  option "with-thread-safety", "Build with thread safety"
+
+  depends_on "autoconf" => :build
+  depends_on "bison" => :build
+  depends_on "re2c" => :build
+  depends_on "libtool" => :run
+  depends_on "aspell"
+  depends_on "argon2"
+  depends_on "curl" if MacOS.version < :lion
+  depends_on "enchant"
+  depends_on "freetds"
+  depends_on "freetype"
+  depends_on "gettext"
+  depends_on "gmp"
+  depends_on "homebrew/apache/httpd24"
+  depends_on "imap-uw"
+  depends_on "icu4c"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libpq"
+  depends_on "mcrypt"
+  depends_on "net-snmp"
+  depends_on "openssl"
+  depends_on "pcre"
+  depends_on "tidy-html5"
+  depends_on "unixodbc"
+  depends_on "webp"
+
+  needs :cxx11
+
+  def install
+    ENV.cxx11
+    config_path = etc/"php/#{version.to_s.split(".")[0..1].join(".")}"
```

Comment:
```
version.to_s.split(".")[0..1].join(".")
```
repeats 7 times times so a function probably makes sense for it.

Reply:
Where should the method be placed compared to the others ?

Reply:
I made a test now, I couldn't replace it in the plist_options since it complained that there was not method with that name.